.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "contrib/poco"]
+	path = contrib/poco
+	url = https://github.com/ClickHouse-Extras/poco
```

Comment:
Are you using 1.8-based Poco? There's a https://github.com/pocoproject/poco/issues/1487 with the older versions (fails to build on OS X). I patched just the file in https://github.com/vavrusa/ClickHouse/commit/7cef1d02e9c45494a802751c3ae8b0d44393e7ff but updating the whole library would be superior.

Reply:
no, poco/develop branch with c++11 support